conf/core/ConfigOptions.asciidoc

```diff
@@ -1160,6 +1168,38 @@ Absolute file path to dictionaries of English words. The first file found will b
 dictionary of English words and used by some algorithms. If the files are not found then the English
 words will be silently ignored.
 
+=== generic.line.tag.threshold
```

Comment:
The generic geometry scripts require both a tag and geometry match. This controls the tag  matching.